av1an-core/src/lib.rs

```diff
@@ -112,6 +112,14 @@ impl Input {
       Input::VapourSynth(path) => vapoursynth::num_frames(path.as_path()).expect(FAIL_MSG),
     }
   }
+
+  pub fn frame_rate(&self) -> f64 {
+    const FAIL_MSG: &str = "Failed to get number of frames for input video";
```

Comment:
Probably just a copy/paste error, but this fail message should probably say failed to get the framerate, instead of number of frames